products/eks/product.yml

```diff
@@ -0,0 +1,33 @@
+product: eks
+full_name:  Amazon Elastic Kubernetes Service
+type: platform
+
+benchmark_id: EKS
+benchmark_root: "../../applications"
+
+profiles_root: "./profiles"
+
+pkg_system: "rpm"
```

Comment:
Is this even needed?

Reply:
apparently :/ ... and it's true. Amazon Linux is based on RHEL (soon on Fedora though), so it's rpm-based lol